packages/gatsby-source-wordpress/src/http-exception-handler.js

```diff
@@ -0,0 +1,26 @@
+const colorized = require(`./output-color`)
```

Comment:
We should drop out this dependency and use the regular colored output

Reply:
👍

Reply:
So no coloring at all? Or use the new reporting stuff @jquense has added?

Reply:
Great! The new stuff of @jquense looks cool